sentry_sdk/integrations/django/transactions.py

```diff
@@ -56,11 +63,23 @@ def _simplify(self, pattern):
         To:
         > "{sport_slug}/athletes/{athlete_slug}/"
         """
+        # "new-style" path patterns can be parsed directly without turning them
+        # into regexes first
+        if DJANGO_VERSION >= (2, 0):
```

Comment:
can we move this check and import to the top and make this branch `if RoutePattern is not None:`